src/components/snapshot/SnapshotAccess.tsx

```diff
@@ -47,7 +47,12 @@ function SnapshotAccess({
 
       // needs this manual conversion because the permissions are different for
       // editing existing snapshot policies vs creating a new snapshot
-      if (role === 'steward' || role === 'reader' || role === 'discoverer') {
+      if (
+        role === 'steward' ||
+        role === 'reader' ||
+        role === 'discoverer' ||
+        role === 'aggregate_data_reader'
+      ) {
```

Comment:
do we want to define an enum of these roles somewhere?

Reply:
Good point - Looking at this reminded me that these components are used in another spot: on snapshot create. That workflow was broken with this change. So, I've now fixed that.